tests/luks-lvm.nix

```diff
@@ -0,0 +1,39 @@
+{ makeTest ? import <nixpkgs/nixos/tests/make-test-python.nix>
+, pkgs ? (import <nixpkgs> {})
+}:
+let
+  makeTest' = args:
+    makeTest args {
+      inherit pkgs;
+      inherit (pkgs) system;
+    };
+  disko-config = import ../example/luks-lvm.nix;
+  tsp-create = pkgs.writeScript "create" ((pkgs.callPackage ../. {}).create disko-config);
+  tsp-mount = pkgs.writeScript "mount" ((pkgs.callPackage ../. {}).mount disko-config);
```

Comment:
Could this be a general test function that we re-ruse?
```
makeDiskoTest = diskoconfig: func: let
     tsp-create = pkgs.writeScript "create" ((pkgs.callPackage ../. {}).create disko-config);
     tsp-mount = pkgs.writeScript "mount" ((pkgs.callPackage ../. {}).mount disko-config);
     args = func { inherit tsp-create, tsp-mount; };
in
     makeTest args {
      inherit pkgs;
      inherit (pkgs) system;
    };
```

Reply:
we probably need to define the number of disks too, but I guess we can generalize that

Reply:
actually we could get the number of disks from the diskoConfig